lib/parser.js

```diff
@@ -147,7 +165,60 @@ function parseClientResponse(msg) {
  * @see https://w3c.github.io/webauthn/#generating-an-attestation-object
  * @see  https://w3c.github.io/webauthn/#defined-attestation-formats
  */
-function parseAttestationObject(attestationObject) {
+/* function parseAttestationObject(attestationObject) {
```

Comment:
Why can we not keep this as deprecated? Can we not make this internally point to the new method? We should try to prevent breaking things.

Reply:
My mistake, I have removed it during test on code coverage and never added again.